gui/hud.gui

```diff
@@ -3555,6 +3730,388 @@ types HUD
 		}
 	}
 
+	type progressbar_hud_infusion = progressbar_hud_stress {
+		size = { 100% 100% }
+
+		using = Animation_ShowHide_Standard
+	}
+
+	type infusion_icon = icon {
+		parentanchor = vcenter
+		size = { 30 30 }
+		framesize = { 70 70 }
+	
+		modify_texture = {
+			visible = "[Or(LessThanOrEqualTo_CFixedPoint(GetPlayer.MakeScope.Var('light_void_infusion_level').GetValue, '(CFixedPoint)-2'), GreaterThanOrEqualTo_CFixedPoint(GetPlayer.MakeScope.Var('light_void_infusion_level').GetValue, '(CFixedPoint)2'))]"
+			name = "lvl3_glow"
+			block "glow_color"
+			{
+				texture = "gfx/interface/colors/gold.dds"
+			}
+			blend_mode = colordodge
+			alpha = 0.2
+		}
+
+		modify_texture = {
+			name = "lvl_increase"
+			texture = "gfx/interface/colors/gold.dds"
+			blend_mode = colordodge
+			alpha = 0
+		}
+
+		state = {
+			name = "stress_level_increase"
+			next = "stress_level_increase_2"
+
+			modify_texture = {
+				name = "lvl_increase"
+				alpha = 0
+			}
+		}
+
+		state = {
+			name = "stress_level_increase_2"
+			next = "stress_level_increase_3"
+			duration = 0.4
+			bezier = { 0.5 0 1 0.5 }
+
+			modify_texture = {
+				name = "lvl_increase"
+				alpha = 0.7
+			}
+		}
+
+		state = {
+			name = "stress_level_increase_3"
+			duration = 0.5
+			bezier = { 0 0.5 0.5 1 }
+
+			modify_texture = {
+				name = "lvl_increase"
+				alpha = 0
+			}
+		}
+	}
+
+	#Warcraft
+	type infusion_bar = widget {
+		name = "infusion_widget"
+		size = { 170 55 }
+		#visible = "[Not(And(EqualTo_int32(FixedPointToInt(GetPlayer.MakeScope.Var('light_void_infusion_level').GetValue), '(int32)0'), EqualTo_int32(FixedPointToInt(GetPlayer.MakeScope.Var('light_void_infusion').GetValue), '(int32)0')))]"
+
+		widget = {
+			parentanchor = center
+			size = { 180 108 }
+
+			tooltip = "PLAYER_INFUSION_TOOLTIP"
+			using = tooltip_ne
+
+			widget = {
+				name = "progressbar"
+				parentanchor = center
+				widgetanchor = left|vcenter
+				position = { -18 0 }
+				size = { 80 20 }
+
+				background = {
+					texture = "gfx/interface/colors/white.dds"
+					color = { 0.1 0.1 0.1 0.8 }
+				}
+
+				hbox = {
+					widget = {
+						layoutpolicy_horizontal = expanding
+						layoutpolicy_vertical = expanding
+						block "stretch_1"
+						{
+							layoutstretchfactor_horizontal = 0
+						}
+
+						progressbar_hud_infusion = {
+							visible = "[GreaterThanOrEqualTo_int32(FixedPointToInt(GetPlayer.MakeScope.Var('light_void_infusion_level').GetValue), '(int32)3')]"
```

Comment:
`light_void` in generic infusion?
P.S. There are more o these, but I don't want to spam.

Reply:
Fixed